tools/DB_date_zeros_removal.php

```diff
@@ -106,11 +106,22 @@
             " SET ".$db->escape($field['COLUMN_NAME'])."=NULL".$autoUpdateSQL.
             " WHERE CAST(".$db->escape($field['COLUMN_NAME'])." AS CHAR(20))='0000-00-00 00:00:00';\n";
     } else {
-	echo "COLUMN ".$field['COLUMN_NAME']." in TABLE ".$field['TABLE_NAME']." is NOT NULLABLE. ".
-	    "A date '1000-01-01' will be entered instead of '0000-00-00' values.\n"; 
-        $nonNullUpdates .= "UPDATE ".$db->escape($database['database']).".".$db->escape($field['TABLE_NAME']).
-            " SET ".$db->escape($field['COLUMN_NAME'])."='1000-01-01'".$autoUpdateSQL.
-            " WHERE CAST(".$db->escape($field['COLUMN_NAME'])." AS CHAR(20))='0000-00-00';\n";
+        echo "COLUMN ".$field['COLUMN_NAME']." in TABLE ".$field['TABLE_NAME']." is NOT NULLABLE. ".
+            "A date '1000-01-01' will be entered instead of '0000-00-00' values.\n";
```

Comment:
you should update this message as it's not accurate in the case of timestamp where you're changing it to 1970-01-01

Reply:
i agree. but what's the reason for the change from '1000-01-01' to '1970-01-01' in the first place?

Reply:
As per @PapillonMcGill's comment: "Datetime and Timestamp have different date range. Timestamp use the (32 bits) unix epoch range"